SergioMendozaParra/August 26/Hackerrank/Java_Inheritance_I/Main.java

```diff
@@ -0,0 +1,10 @@
+package java_inheritance1;
```

Comment:
I think underscore it is allowed but... I don't recommended it in package names
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html